### ISSUE TITLE
Feature request: First run wizard.

### ISSUE THREAD

**Issue by Insti** [exercism/gui#45]
When I first start the app, I worked out I needed to put in my API key, did that and hit save, I'm still on the configuration screen. What do I do next? (This is not a support request)

There needs to be a first run wizard that prompts for an API key and which language you're interested in then takes you to the relevant track.

**Comment by holandes22** [comment]
I think this is great feedback. I added to the home page a "get started" button that takes you to the config page if not configured or to the tracks route if it is configured. But you are right, once configured would be great to indicate that the next step is to go "tracks" or a specific track.

**Comment by Insti** [comment]
Sorry, mis-clicked close.

& ideally the API key could fetch itself.

**Comment by holandes22** [comment]
Yeah fetching the API key automatically is indeed tricky. It will require the exercism API to support login via OAuth or other third party and then implementation on the client. Is a good idea, but out of the scope of this feature request I think

**Comment by holandes22** [comment]
I really like this idea, but unfortunately I don't have any bandwidth during August. Will definitely work on adding this as soon as I can

**Comment by alebaffa** [comment]
I just downloaded the app and it found the API key by itself fetching it from my local .exercism.json file (I'm on Windows 10 x64). Is it the same thing you're referring to, @Insti ?

**Comment by Insti** [comment]
@alebaffa, not exactly the same, but it might be part of it. 
I didn't already have an .exercism.json file
